azure-spring-boot/src/test/java/com/microsoft/azure/spring/autoconfigure/sqlserver/AlwaysEncryptedPostProcessorTest.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.autoconfigure.sqlserver;
+
+import org.junit.Test;
+import org.mockito.Mock;
+import org.springframework.core.env.ConfigurableEnvironment;
+import java.util.Map;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class AlwaysEncryptedPostProcessorTest {
+
+    @Mock ConfigurableEnvironment environment;
+
+
+    @Test
+    public void dataSourcePropertiesEncryptionEnabled() {
+        environment = mock (ConfigurableEnvironment.class);
```

Comment:
This is unnecessary since `@Mock` will do the same thing.

Reply:
not really, initialization still required. getting null pointer if removed